test/PullPayment.js

```diff
@@ -1,15 +1,19 @@
 var PullPaymentMock = artifacts.require("./helpers/PullPaymentMock.sol");
 
 contract('PullPayment', function(accounts) {
-
+  let ppce;
+  let tAMOUNT = 17*1e18;
```

Comment:
To differentiate of the local variable used in the second test case. Yes, the name can be changed to be more descriptive. `t` came from transaction.